sdk/hdinsight/Microsoft.Azure.Management.HDInsight/src/Microsoft.Azure.Management.HDInsight.csproj

```diff
@@ -7,14 +7,12 @@
 		<PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
 		<Description>Azure HDInsight Management SDK Library</Description>
 		<AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
-		<Version>5.1.0</Version>
+		<Version>5.2.0</Version>
```

Comment:
For hotfix the patch (third) version should be updated, i.e. `5.1.1`

Reply:
> For hotfix the patch (third) version should be updated, i.e. `5.1.1`

Fixed. Thanks